### PR TITLE
Update libraries.json to only include 1 library key

### PR DESCRIPTION
**Pull request by mborland**
Not too long ago this was asked about on the list, and revitalized on slack yesterday. @jzmaddock, @ckormanyos  and @NAThompson is there anything you want to add to the description? I hit the major sections that are in our readme.

### CODE REVIEW

**Comment by ckormanyos** [comment]
> anything you want to add to the description?

I've always been so impressed with seamless interaction (template scalability) of special functions and math tools like root finding. In fact their scalability to the extended width types in Multiprecision is a key fundamental excellence of Math.

Is there any way to express this - in a more terse and clear way than my roundabout mention here?

I think it's worth mentioning in the top-level summary.

Cc: @jzmaddock and @mborland and @NAThompson

**Comment by mborland** [comment]
> I've always been so impressed with seamless interaction (template scalability) of special functions and math tools like root finding. In fact their scalability to the extended width types in Multiprecision is a key fundamental excellence of Math.

I concur and have added a comment about it.

**Comment by ckormanyos** [comment]
>> scalability ... is a key fundamental excellence of Math.

> I concur and have added a comment about it.

Awesome. I like the terse, clear way you expressed that. Thank you Matt

File: meta/libraries.json [hunk 1]
```diff
@@ -5,82 +5,15 @@
...
+        "description": "Boost.Math includes several contributions in the domain of mathematics: Floating Point Utilities, Specific Width Floating Point Types, Mathematical Constants, Statistical Distributions, Special Functions, Root Finding and Function Minimization, Polynomials and Rational Functions, Interpolation, Numerical Integration and Differentiation, Quaternions and Octonions. Many of these features are templated to support both built-in, and extended width types (e.g. Boost.Multiprecision)",
```
[the remark this answers is not in the document]
Comment (mborland) [reply meta/libraries.json:8]: I think that is fair. QVM is still getting regular updates.

**Pull request merged**